server/src/util/autocomplete.ts

```diff
@@ -166,6 +169,20 @@ export function getAttributeAutocomplete(options: IAutocompleteArguments): Compl
 
 
     return generateAutocomplete(attributeObjects);
+
+}
+
+export function getAttributeAutocomplete(options: IAutocompleteArguments): CompletionList {
+    switch(options.scopeAtPos.tagName) {
+        case 'style':
+                const languageService = getLESSLanguageService();
```

Comment:
So the default for a style tag should use css (not less). We should read the extension on the stylesheet (`style.less`, `style.scss`, `style.stylus`) and hopefully we could have a map of all of these different supported languages.

Also, this is going to be awesome!

Reply:
This is actually for inside the marko template for inline styles to autocomplete. I assume that the default syntax for the marko file for styles is less, is that correct? Can it be something else.
For normal less styles it should be picked up automatically and this code should not be checked.